client/src/services/services.js

```diff
@@ -10,6 +11,7 @@ const storage = new Storage({
 });
 
 const http = new Http({
+  apiPath: ENV.API_PATH,
```

Comment:
FYI, same about prefix

Reply:
the "path" is okay, because we can pass any link (not just the prefix)